subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -1840,3 +2060,11 @@ struct proc_ctx *llcp_create_procedure(enum llcp_proc proc)
 	return create_procedure(proc, &mem_local_ctx);
 }
 #endif
+
+bool phy_valid(uint8_t phy)
+{
+	/* This is equivalent to:
+	 * exactly one bit set, and no bit set is rfu's
+	 */
+	return (phy == PHY_1M || phy == PHY_2M || phy == PHY_CODED);
```

Comment:
Should `PHY_2M` be considered valid if `CONFIG_BT_CTLR_PHY_2M=n`? Ditto with coded?

Reply:
It will be checked after: see: `ull_llcp_past.c` line:83:

`#if defined(CONFIG_BT_CTLR_PHY)
	const uint8_t phy = pdu->llctrl.periodic_sync_ind.phy;

	if (((phy & PHY_2M) && !IS_ENABLED(CONFIG_BT_CTLR_PHY_2M)) ||
	    ((phy & PHY_CODED) && !IS_ENABLED(CONFIG_BT_CTLR_PHY_CODED))) {
		/* Unsupported phy selected */
		return BT_HCI_ERR_UNSUPP_FEATURE_PARAM_VAL;
	}
#endif /* CONFIG_BT_CTLR_PHY */`